app/controllers/api/v1/slides_controller.rb

```diff
@@ -1,21 +1,21 @@
 class Api::V1::SlidesController < ApplicationController
-  before_action :set_slide, only: %i[show update destroy]
   before_action :authorize_request
+  before_action :set_slide, only: [:show, :update, :destroy]
```

Comment:
Es valido utilizar `%i` en un array de symbols:
```suggestion
  before_action :set_slide, only:  i%[show update destroy]
```